src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -417,11 +417,14 @@ private function getNewOrder(array $uses, Tokens $tokens)
     private function sortByAlgorithm($indexes)
     {
         switch ($this->sortAlgorithm) {
+            case self::SORT_ALPHA:
+                uasort($indexes, array($this, 'sortAlphabetically'));
+                break;
             case self::SORT_LENGTH:
                 uasort($indexes, array($this, 'sortByLength'));
                 break;
             default:
-                uasort($indexes, array($this, 'sortAlphabetically'));
+                throw new InvalidFixerConfigurationException('ordered_use', sprintf('Sort algorithm is invalid. Should be one of the: "%s"', implode('", "', $this->supportedSortAlgorithms)));
```

Comment:
here, it's not setting up the onfiguration so this exception is overused.
just rise logic exception